backend/src/test/de.grilborzer.neuefische.sandwichmaker.controller/SandwichControllerTest.java

```diff
@@ -0,0 +1,121 @@
+package de.grilborzer.neuefische.sandwichmaker.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.grilborzer.neuefische.sandwichmaker.model.Sandwich;
+import de.grilborzer.neuefische.sandwichmaker.repository.SandwichRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class SandwichControllerTest {
+
+    @Autowired
+    MockMvc mockMvc;
+
+    @Autowired
+    ObjectMapper objectMapper;
+
+    @Autowired
+    SandwichRepository sandwichRepository;
+
+    final String sandwichUrl = "/api/sandwich/";
+
+    final Sandwich testSandwichWithId = Sandwich.builder()
```

Comment:
Im always a bit hesitated about using predefined test-object for all test cases. But why not? 🤔